docs/yaml/functions/custom_target.yaml

```diff
@@ -31,7 +31,9 @@ description: |
   - `@OUTDIR@`: the full path to the directory where the output(s) must be written
   - `@DEPFILE@`: the full path to the dependency file passed to `depfile`
   - `@PLAINNAME@`: the input filename, without a path
+  - `@PLAINNAME0@` `@PLAINNAME1@` `...`: the input filename without a path, with the specified array index in `input`
```

Comment:
Need the `*(since 1.5.0)*` in the custom_target docs, otherwise this looks good to me.

Reply:
Done.